src/Proxy.NET/Routes/ProxyRoutes.cs

```diff
@@ -0,0 +1,10 @@
+namespace Proxy.NET.Routes;
+
+public static class RouteExtensions
+{
+    public static IEndpointRouteBuilder MapProxyRoutes(this IEndpointRouteBuilder builder)
+    {
+        builder.MapGroup("/api/v1").MapAttendeeRoutes().MapEventRoutes().MapAzureOpenAIRoutes().MapAzureAISearchRoutes();
+        return builder;
+    }
```

Comment:
```suggestion
    public static IEndpointRouteBuilder MapProxyRoutes(this IEndpointRouteBuilder builder) =>
        builder
        .MapGroup("/api/v1")
        .MapAttendeeRoutes()
        .MapEventRoutes()
        .MapAzureOpenAIRoutes()
        .MapAzureAISearchRoutes();
```

Just a little cleaner